lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -1376,27 +1376,27 @@ handle_terminator(Rest, Line, Column, Scope, Token, Tokens) ->
       tokenize(Rest, Line, Column, New, [Token | Tokens])
   end.
 
-check_terminator({Start, {Line, _, _}}, Terminators, Scope)
+check_terminator({Start, {Line, Column, _}}, Terminators, Scope)
```

Comment:
Instead of picking the Line and Column, you can include the whole metadata instead:

```suggestion
check_terminator({Start, Meta}, Terminators, Scope)
```

And put the whole `Meta` in the terminator. WDYT?

Reply:
Are these changes what you have in mind? [c901624](https://github.com/elixir-lang/elixir/pull/12916/commits/c90162427eb961846cfb43e6ffdbcb828af4397a)

Not 100% sure if the indentation should be included in meta, but I believe not